src/change_stream/options.rs

```diff
@@ -0,0 +1,79 @@
+//! Contains options for ChangeStreams.
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+use std::time::Duration;
+use typed_builder::TypedBuilder;
+
+use crate::{change_stream::event::ResumeToken, collation::Collation, options::AggregateOptions};
+
+/// These are the valid options that can be passed to the `watch` method for creating a
+/// [`ChangeStream`](crate::change_stream::ChangeStream).
```

Comment:
We can also remove this link (and others in this file)

Reply:
Not sure why?

Reply:
oh sorry misread, the types aren't in scope so we do need the links